src/dtls13.c

```diff
@@ -2842,5 +2842,15 @@ int wolfSSL_dtls13_allow_ch_frag(WOLFSSL *ssl, int enabled)
 }
 #endif
 
+#ifdef WOLFSSL_DTLS13_NO_HRR_ON_RESUME
+int wolfSSL_dtls13_hrr_on_resume(WOLFSSL *ssl, int enabled)
```

Comment:
The name of the API is misleading, AFAIU if enabled is true then *no* hrr on resume happens.

Reply:
Fixing.